houston/generator/__init__.py

```diff
@@ -0,0 +1,2 @@
+from houston.generator.rand import RandomMissionGenerator
```

Comment:
Minor comment: from now onwards, prefer relative imports (e.g., `from .rand import RandomMissionGenerator`). Also, if you're adding or modifying an existing module, add `__all__` to prevent pollution of the namespace when using `from`.